nimbus_nlp/models/features/overall_features.json

```diff
@@ -1 +1 @@
-{"[COURSE]": 0, "are": 0, "what": 0, "prereqs": 0, "required": 0, "which": 0, "course": 0, "require": 0, "take": 0, "count": 0, "how": 0, "many": 0, "unit": 0, "towards": 0, "offered": 0, "when": 0, "typically": 0, "offer": 0, "is": 0, "quarter": 0, "going": 0, "go": 0, "next": 0, "Is": 0, "4": 0, "class": 0, "term": 0, "prerequisite": 0, "have": 0, "recommend": 0, "need": 0, "order": 0, "covered": 0, "topic": 0, "cover": 0, "title": 0, "called": 0, "also": 0, "call": 0, "name": 0, "code": 0, "time": 0, "per": 0, "year": 0, "available": 0, "fulfill": 0, "Are": 0, "preqs": 0, "'s": 0, "soon": 0, "description": 0, "learn": 0, "-PRON-": 0, "needed": 0, "much": 0, "credit": 0, "worth": 0, "number": 0, "corequisit": 0, "concurrent": 0, "enrollement": 0, "listed": 0, "list": 0, "lab": 0, "[PROF]": 0, "research": 0, "area": 0, "email": 0, "phone": 0, "interest": 0, "interested": 0, "researching": 0, "contact": 0, "information": 0, "reach": 0, "where": 0, "office": 0, "room": 0, "find": 0, "website": 0, "personal": 0, "home": 0, "page": 0, "address": 0, "job": 0, "lecturer": 0, "professor": 0, "position": 0, "cal": 0, "poly": 0, "building": 0, "[CLUB]": 0, "club": 0, "box": 0, "affiliated": 0, "college": 0, "affiliate": 0, "president": 0, "advisor": 0, "do": 0, "who": 0, "chief": 0, "adviser": 0, "usually": 0, "currently": 0, "good": 0, "taught": 0, "teach": 0, "[SECRET_HIDEOUT]": 0, "whose": 0, "In": 0, "does": 0, "event": 0, "week": 0, "platform": 0, "hour": 0, "virtual": 0, "zoom": 0, "vroom": 0, "having": 0, "skype": 0, "not related": 0}
\ No newline at end of file
+{"prereqs": 0, "COURSE": 0, "be": 0, "course": 0, "require": 0, "take": 0, "unit": 0, "count": 0, "typically": 0, "offer": 0, "quarter": 0, "go": 0, "4": 0, "class": 0, "term": 0, "prerequisite": 0, "have": 0, "recommend": 0, "need": 0, "order": 0, "topic": 0, "cover": 0, "title": 0, "call": 0, "code": 0, "time": 0, "year": 0, "available": 0, "fulfill": 0, "preqs": 0, "soon": 0, "description": 0, "learn": 0, "credit": 0, "worth": 0, "number": 0, "corequisite": 0, "concurrent": 0, "enrollement": 0, "list": 0, "lab": 0, "PROF": 0, "research": 0, "area": 0, "email": 0, "phone": 0, "interest": 0, "interested": 0, "contact": 0, "information": 0, "reach": 0, "office": 0, "room": 0, "find": 0, "website": 0, "personal": 0, "home": 0, "page": 0, "address": 0, "job": 0, "lecturer": 0, "professor": 0, "position": 0, "Cal": 0, "Poly": 0, "building": 0, "CLUB": 0, "club": 0, "box": 0, "college": 0, "affiliate": 0, "President": 0, "Advisor": 0, "do": 0, "chief": 0, "adviser": 0, "usually": 0, "currently": 0, "good": 0, "teach": 0, "advisor": 0, "build": 0, "in": 0, "event": 0, "week": 0, "platform": 0, "hour": 0, "virtual": 0, "zoom": 0, "vroom": 0, "Zoom": 0, "Skype": 0, "see": 0, "not related": 0}
```

Comment:
I'm going to assume this dict is generated somewhere else and was just updated or something

Reply:
I think it's generated by training the model, I wonder if @richagadgil or @cameron-toy have any insight about why it's different.